README.md

```diff
@@ -89,77 +89,92 @@ WSO2 API Kubernetes Platform has released following docker images in the WSO2 pu
 ### Before you begin...
 
 * Install [Helm](https://helm.sh/docs/intro/install/)
```

Comment:
Shall we add the helm version as 3.11.x?

Reply:
Added by https://github.com/wso2/apk/pull/1104/commits/a6d95d94705d687490dfd85ed65053bb2a0c6473